src/zkbob/manager/kyc/SimpleKYCProviderManager.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: CC0-1.0
+
+pragma solidity 0.8.15;
+
+import "../../../interfaces/IKycProvidersManager.sol";
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+
+/**
+ * @title SimpleKYCProviderManager
+ * @dev Implements KYC verifier based on specific NFT ownership.
+ */
+contract SimpleKYCProviderManager is IKycProvidersManager {
+    // ownership of a token in this NFT-contract will be checked to consider an account as passed KYC
+    IERC721 public immutable NFT;
+    // this tier number will be used for all users passed KYC
+    uint8 immutable tierForPassedKYC;
+
+    constructor(IERC721 _token, uint8 _tier) {
+        require(address(_token) != address(0), "KYCProviderManager: token address is zero");
+        NFT = _token;
+        tierForPassedKYC = _tier;
+    }
+
+    function passesKYC(address _user) external view override returns (bool) {
```

Comment:
Seems that now this is the same as:
```solidity
(bool ok, ) = manager. getIfKYCpassedAndTier(_user);
```
Do you really think we need a separate method for such request?

Reply:
Addressed in https://github.com/zkBob/zkbob-contracts/pull/54/commits/714ee5cd9b313b7dbee0c1b18172af59ce7a8a1f